maintainers/maintainer-list.nix

```
@@ -58,6 +58,10 @@
          nix-build lib/tests/maintainers.nix
 
     See `./scripts/check-maintainer-github-handles.sh` for an example on how to work with this data.
+
+    When creating a commit:
+      * When adding a new maintainer, be aware of the current commit conventions:
+        https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions.
```

```suggestion

    When adding a new maintainer, be aware of the current commit conventions documented at CONTRIBUTING.md in the root of Nixpkgs repo.
```

Opening a web browser in order to read a local file is somewhat convoluted.

---

The URL conveys the same information (plus anchor) while also providing a handy link if you're not currently sitting inside an editor.

---

> while also providing a handy link if you're not currently sitting inside an editor.

Like, huh, a web browser?

But your argument has some basis. Let's reword it.